Extensions/show_more.js

```diff
@@ -645,8 +618,8 @@ XKit.extensions.show_more = new Object({
 
 				var username = $(this).parent().attr('data-tumblelog-name');
 
-				if (XKit.extensions.show_more.submit_available[username] === true ||
-					XKit.extensions.show_more.submit_available[username] === false) {
+				if (XKit.extensions.show_more.submit_available[username] ||
+					!XKit.extensions.show_more.submit_available[username]) {
```

Comment:
Good catch. To be safe, I'll change them to `if (typeof(XKit.extensions.show_more.submit_available[username]) === "boolean")` as well as elsewhere in this file.